.github/workflows/e2e.yml

```diff
@@ -30,14 +30,14 @@ jobs:
 
     - name: Build docker
       run: |
-        docker build -t ghcr.io/mneverov/cluster-cidr-controller:ci-e2e-test -f Dockerfile .
+        docker build -t ghcr.io/sigs.k8s.io/node-ipam-controller:ci-e2e-test -f Dockerfile .
```

Comment:
Can we use GCP GCR instead even local/step builds ? GHCR is currently not enabled

Reply:
I think you need to give some directions Arnaud on how to do it

Reply:
my bad. I understand that it is possible to use https://github.com/kubernetes/registry.k8s.io, right?
Also, my understanding that it is not `ghcr.io/sigs.k8s.io/node-ipam-controller` but `ghcr.io/kubernetes-sigs/node-ipam-controller` and it should work, correct?

Reply:
Should be `gcr.io/k8s-staging-networking/node-ipam-controller`. 
Also push the image to a registry can't be currently done with Github Actions.

Reply:
@ameukam my understanding is that this [PR](https://github.com/kubernetes/test-infra/pull/31636) is a prerequisite to this, right?